configure.ac

```diff
@@ -65,6 +65,12 @@ if test "x$with_meu" != "xno"; then
 	fi
 fi
 
+#check if we should enable debug build
+AC_ARG_ENABLE(debug, [AS_HELP_STRING([--enable-debug],[enable debug, default: no])], enable_debug=$enableval, enable_debug=no)
+if test "$enable_debug" = "yes"; then
+	AC_DEFINE([DEBUG_BUILD], [1], [Configure to build debug version])
+fi
+
```

Comment:
if this enough? Or should the tags be in a part of the firmware file that can be stripped out to compare files?